### PR TITLE
Code added for Issue Dezenix#33 10Doc by Arpan Sharma

### PR DESCRIPTION
**Pull request by arpansharma-11**
This PR is for the assigned issue Dezenix#33 10DoC Hover UI Effect.

### CODE REVIEW

File: UI HOVER EFFECT/code/style.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+
+img {
+    border-radius: 50%;
+    height: 130px;
+    width: 130px;
+    margin-top: -50%;
+    padding-left: 10px;
+}
```
Comment (saikatmu31) [inline UI HOVER EFFECT/code/style.css:13]: Avoid Targeting the Elements by HTML tags instead use some class or ID

**Review by saikatmu31 — CHANGES_REQUESTED** [review]
Also add a readme file

**Comment by Abirpal202049** [comment]
Yes make the changes as mentioned by @saikatmu31

File: UI HOVER EFFECT/code/style.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
+
+#content{
+    background-color: #f3f1fe;
+    align-items: center;
+
```
Comment (Ovenoboyo) [inline UI HOVER EFFECT/code/style.css:5]: Please remove extra whitespaces inside and after css classes. For styling guidelines you can refer to [this](https://google.github.io/styleguide/htmlcssguide.html)

File: UI HOVER EFFECT/code/tilt.js [hunk 3]
```diff
@@ -0,0 +1,183 @@
+var VanillaTilt = function () {
+    "use strict";
+    class t {
```
Comment (Ovenoboyo) [inline UI HOVER EFFECT/code/tilt.js:3]: Please avoid making spaghetti by declaring a class inside a function then instantiating and returning it in an anonymous function. Write clean code by separating all components and write comments such that it clearly shows the flow of your code.

File: UI HOVER EFFECT/code/tilt.js [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+            this.width = null, this.height = null, this.clientWidth = null, this.clientHeight = null, this.left = null, this.top = null, this.gammazero = null, this.betazero = null, this.lastgammazero = null, this.lastbetazero = null, this.transitionTimeout = null, this.updateCall = null, this.event = null, this.updateBind = this.update.bind(this), this.resetBind = this.reset.bind(this), this.element = e, this.settings = this.extendSettings(i), this.reverse = this.settings.reverse ? -1 : 1, this.glare = t.isSettingTrue(this.settings.glare), this.glarePrerender = t.isSettingTrue(this.settings["glare-prerender"]), this.fullPageListening = t.isSettingTrue(this.settings["full-page-listening"]), this.gyroscope = t.isSettingTrue(this.settings.gyroscope), this.gyroscopeSamples = this.settings.gyroscopeSamples, this.elementListener = this.getElementListener(), this.glare && this.prepareGlare(), this.fullPageListening && this.updateClientSize(), this.addEventListeners(), this.reset(), this.updateInitialPosition()
+        }
+        static isSettingTrue(t) {
+            return "" === t || !0 === t || 1 === t
+        }
+        getElementListener() {
```
Comment (Ovenoboyo) [inline UI HOVER EFFECT/code/tilt.js:11]: also make use of [private class methods and properties](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields) if your method or property isn't going to be referenced from outside the class

File: UI HOVER EFFECT/code/tilt.js [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+                    reset: !0,
+                    gyroscope: !0,
+                    gyroscopeMinAngleX: -45,
+                    gyroscopeMaxAngleX: 45,
+                    gyroscopeMinAngleY: -45,
+                    gyroscopeMaxAngleY: 45,
+                    gyroscopeSamples: 10
+                },
+                i = {};
+            for (var s in e)
```
Comment (Ovenoboyo) [inline UI HOVER EFFECT/code/tilt.js:164]: Please prefer using {}. Even though not using them is syntactically correct, it usually makes the code difficult to read and understand

File: UI HOVER EFFECT/code/tilt.js [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+                    startX: 0,
+                    startY: 0,
+                    perspective: 1e3,
+                    easing: "cubic-bezier(.03,.98,.52,.99)",
+                    scale: 1,
+                    speed: 300,
+                    transition: !0,
+                    axis: null,
+                    glare: !1,
+                    "max-glare": 1,
```
Comment (Ovenoboyo) [inline UI HOVER EFFECT/code/tilt.js:151]: avoid using "-" when defining properties in a JS object. Instead prefer using "_"

**Review by Ovenoboyo — CHANGES_REQUESTED** [review]
If script.js is empty and unnecessary, please delete it. And please use clear folder naming conventions. For example you may put all your javascript code in a folder called "js", your css in a folder called "css" instead of putting it all inside "code"

**Pull request closed without merging**